doc/_templates/panelitelink.html

```diff
@@ -0,0 +1,8 @@
+{# Custom template to include a link to Panelite #}
+{% if title and pagename != root_doc %}
+  <div class="tocsection">
+    <a href="{{ panelite_endpoint + '/lab?path=' + pagename + '.ipynb' }}" target=”_blank”>
```

Comment:
`pagename` includes the path, e.g. `how_to/styling/design`?

Reply:
That's my observation.

Reply:
👍